app/invocation/invocation.tsx

```diff
@@ -209,16 +213,9 @@ export default class InvocationComponent extends React.Component<Props, State> {
     });
     const isBazelInvocation = this.state.model.isBazelInvocation();
     const fetchBuildLogs = () => {
-      return rpcService.service
-        .getEventLogChunk(
-          new eventlog.GetEventLogChunkRequest({
-            invocationId: this.props.invocationId,
-            minLines: maxInt32, // int32 max value: max number of lines we can request.
-          })
-        )
-        .then((response: eventlog.GetEventLogChunkResponse) => {
-          return new TextDecoder().decode(response.buffer || new Uint8Array());
-        });
+      rpc_service.downloadBytestreamFile("//buildlog", "bytestream://", this.props.invocationId, {
```

Comment:
Since the bytestream API is only used to fetch things from cache (CAS/AC), and build logs are stored in Blobstore (not cache), I think it'd be clearer to add a separate function to RPCService called `downloadBlob` which fetches from Blobstore .

It could still use the same `/file/download` endpoint, but just add a separate URL param like `?blobtype=buildlog` or something like that, which tells the file download handler to fetch the invocation build log from blobstore.

Reply:
sure, fine with me.